py_gtfs_rt_ingestion/tests/test_convert.py

```diff
@@ -15,52 +15,54 @@ def test_vehicle_positions_file_conversion(tmpdir):
 
     table = convert_files(filepaths=[rt_vehicle_positions_file],
                           config=config)
-
-    # these are the columns one would expect from looking at the json data
-    paths_from_json = set([
-        "entity_id",
-        "current_status",
-        "current_stop_sequence",
-        "occupancy_percentage",
-        "occupancy_status",
-        "stop_id",
-        'vehicle_timestamp',
-        "bearing",
-        "latitude",
-        "longitude",
-        "speed",
-        "direction_id",
-        "route_id",
-        "schedule_relationship",
-        "start_date",
-        "start_time",
-        "trip_id",
-        "vehicle_id",
-        "vehicle_label",
-        "vehicle_consist",
-    ])
-
-    # these are the columns that are added in for the header
-    paths_from_header = set([
-        "feed_timestamp",
-        "year",
-        "month",
-        "day",
-        "hour",
-    ])
-
-    all_expected_paths = paths_from_json | paths_from_header
-
-    found_paths = set()
-    for item in table.schema.names:
-        found_paths.add(item)
+    np_df = table.to_pandas()
+
+    # tuple(na count, dtype, min, max)
+    file_details = {
+        'year':(0,'int16','2022','2022'),
+        'month':(0,'int8','1','1'),
+        'day':(0,'int8','1','1'),
+        'hour':(0,'int8','0','0'),
+        'feed_timestamp':(0,'int64','1640995202','1640995202'),
+        'entity_id':(0,'object','y4124','1625'),
+        'current_status':(0,'object','STOPPED_AT','INCOMING_AT'),
+        'current_stop_sequence':(29,'float64','640.0','1.0'),
+        'occupancy_percentage':(426,'float64','nan','nan'),
+        'occupancy_status':(190,'object','nan','nan'),
+        'stop_id':(29,'object','nan','nan'),
+        'vehicle_timestamp':(0,'int64','1640995198','1640994366'),
+        'bearing':(0,'int64','360','0'),
+        'latitude':(0,'float64','42.778499603271484','41.82632064819336'),
+        'longitude':(0,'float64','-70.7895736694336','-71.5481185913086'),
+        'speed':(392,'float64','29.9','2.6'),
+        'direction_id':(4,'float64','1.0','0.0'),
+        'route_id':(0,'object','Shuttle-Generic','1'),
+        'schedule_relationship':(0,'object','UNSCHEDULED','ADDED'),
+        'start_date':(40,'object','nan','nan'),
+        'start_time':(87,'object','nan','nan'),
+        'trip_id':(426,'object','nan','nan'),
+        'vehicle_id':(0,'object','y4124','1625'),
+        'vehicle_label':(0,'object','4124','0420'),
+        'vehicle_consist':(324,'object','nan','nan'),
+    }
```

Comment:
There's definitely a better way to do this, but this dictionary has each Table column as a key and then count of (na values in rows, dtype, min and max) values stored in a tuple. 

Not all columns can have min/max values, those have 'nan' stored for min/max.